src/components/EntityPage/File/index.js

```diff
@@ -206,6 +210,94 @@ const fileQuery = `query ($sqon: JSON) {
   }
 }`;
 
+const summaryTable1Data = data => {
+  const participants = data.participants.hits.edges[0].node;
+  const study = participants.study;
+  const biospecimens = participants.biospecimens.hits.edges[0].node;
+  return [
+    {
+      title: 'External ID:',
+      summary: <ExternalLink href={``}>data.external_id</ExternalLink>,
+    },
+    { title: 'Name:', summary: data.file_name },
+    { title: 'Study:', summary: `${study.short_name}(${study.kf_id})` },
+    { title: 'Access:', summary: data.controlled_access ? 'Controlled' : '' },
+    { title: 'Consent Codes:', summary: biospecimens.dbgap_consent_code },
+  ];
+};
+
+const formatToGB = size => `${size / 1000000} GB`; // TODO: What size is coming back from API?
```

Comment:
@ciaranschutte Not sure either if that bits or bytes, but there is a difference in calculation https://www.gbmb.org/kb-to-gb

Reply:
@nyanofthemoon yup. Just put in _a_ calculation to display something :)